src/api/common/schemas/get-paths.json

```diff
@@ -6,7 +6,23 @@
     "type": "object",
     "properties": {
       "source": {"$ref": "adjustment"},
-      "destination": {"$ref": "adjustment"},
+      "destination": {
+        "type": "object",
```

Comment:
Oh, I thought the wildcard form only applied to pathfinding, but I just noticed the docs say it works for payments. Maybe the right answer is to parse the metadata to figure out the actual counterparty?